conda/cli/helpers.py

```diff
@@ -449,7 +450,7 @@ def add_parser_package_install_options(p: ArgumentParser) -> _ArgumentGroup:
         "Package Linking and Install-time Options"
     )
     package_install_options.add_argument(
```

Comment:
The deprecation you want is `25.9` and `26.3` (I just opened a PR to add the deprecation formula to the [docs](https://github.com/conda/conda/pull/14447)).

We also have a `deprecated.action` helper:

```suggestion
    package_install_options.add_argument(
        "-f",
        dest="force",
        action=deprecated.action(
            "25.9",
            "26.3",
            argparse._StoreTrueAction,
            addendum="Use `--force` instead.",
        ),
        default=False,
        help=SUPPRESS,
    )
    package_install_options.add_argument(
```

Reply:
15 months for this change seems too long in this specific case. This has been ˝functionally deprecated" (suppressed) for a long time now, so I thought we could skip the PendingDeprecation stage and kick the six-month period with 25.1.

Also this message is not accurate because `--force` does not have any effect anymore; users need to resort to `--force-reinstall` or `--yes`, depending on the intended semantics.

Reply:
1. it sounds like we should be deprecating `--force` not just trying to deprecate `-f`
2. if we feel that strongly about deprecating this we can go with 25.3 and 25.9?

Reply:
About (1), I think we are doing two things: deprecating _some_  uses of `--force` (with install and update, but not with remove) and repurposing `-f`.

About (2), I still think 25.1 and 25.7 is generous enough, but I'll defer that to the broader team.

Reply:
I don't think we can soften the deprecation policy, since then, we start debating every deprecation, 2026 sounds far, but it's not really, it's just 12.5 months away

Reply:
I would agree if `conda install -f numpy` had a different outcome than `conda install numpy` but these days it doesn't. Folks using `-f` would get a warning about the upcoming meaning change for six months, and I feel that's sufficient notice. 

I just want to avoid the weird time window where users can do `mamba create -f env.yml` but have to do `conda create --file env.yml` only to end up learning that six months later they'll be able to do `conda create -f env.yml` too.